### PR TITLE
added acount voor peter van eynde

### PR DESCRIPTION
**Pull request by corosuske**

### CODE REVIEW

**Comment by SuperQ** [comment]
no ssh key?

**Comment by RichiH** [comment]
We should all use ssh keys, only.

Richard

Sent by mobile; excuse my brevity.

**Comment by corosuske** [comment]
I was looking into where to host peters key (it was added to the mail) ...

Then my wife came with food

Will add the key soon

**Comment by SuperQ** [comment]
You can include keys locally in the repo if necessary, see the Documentation[0]

[0]: http://docs.ansible.com/ansible/authorized_key_module.html

**Comment by corosuske** [comment]
added the key ( i think) 
please verify i dont normally work with ansible

**Comment by SuperQ** [comment]
the lookup method you used is to copy a file that already exists on the node.

You probably want
```
- authorized_key: user=pevaneyn key="{{ item }}"
  with_file:
    - authorised_keys/pvaneynde.pub
```

File: ansible/roles/common/files/authorised_keys/pvaneynde.pub [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHjErAEeV7XmajochqXCGqu3YaLKsfNnCksX/WcySRZ6 pevaneyn@PEVANEYN-M-50LF
```
Comment (SuperQ) [inline ansible/roles/common/files/authorised_keys/pvaneynde.pub]: You don't want to split roles like this, keys should be in `ansible/roles/users/files/foo.pub`

**Comment by corosuske** [comment]
i think i fixed it now

**Comment by SuperQ** [comment]
I've applied this config to the server

**Pull request merged**